modules/valkey/options.go

```diff
@@ -0,0 +1,82 @@
+package valkey
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+
+	"github.com/mdelapenya/tlscert"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+type options struct {
+	tlsEnabled bool
+	tlsConfig  *tls.Config
+}
+
+// Compiler check to ensure that Option implements the testcontainers.ContainerCustomizer interface.
+var _ testcontainers.ContainerCustomizer = (Option)(nil)
+
+// Option is an option for the Redpanda container.
+type Option func(*options) error
+
+// Customize is a NOOP. It's defined to satisfy the testcontainers.ContainerCustomizer interface.
+func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
+	// NOOP to satisfy interface.
+	return nil
+}
```

Comment:
suggestion: not needed for this PR but as this is a very common pattern we should create a noop customiser we can just embed to avoid this repeating boilerplate.

Reply:
Makes sense! Thanks for the suggestion!